tests/test_pattern.py

```diff
@@ -11,6 +11,7 @@
 
 import graphix.clifford
 import graphix.ops
+import graphix.parameter
```

Comment:
should we add option to get parameterized (say replace a small number of angles with placeholder) circuits in `test.random_circuit.get_rand_circuit`, so that we can add parameterized version of patterns in the existing test suite?

Reply:
Fixed in commit 05c05a0. These tests highlight that `Placeholder` objects without any form of computation were not suitable for use in rotation gates, as angles in rotation gates are transformed by the transpiler into expressions like `-alpha / pi`. This observation is related to @EarlMilktea's remark about simple computations such as sign flips and constant shifts. The commit 05c05a0 adds support for affine computation, which is simple enough to maintain while allowing these necessary operations.